tools/hifiasm/hifiasm.xml

```diff
@@ -522,36 +532,45 @@
                 </assert_contents>
             </output>
         </test>
+        <!-- TEST 14: test bin files -->
+        <test expect_num_outputs="6">
+            <param name="reads" value="hifiasm-in1.fa.gz" ftype="fasta.gz" />
+            <param name="filter_bits" value="0" />
+            <param name="mode_selector" value="standard" />
+            <param name="bins_out" value="yes" />
+            <output name="ovlp_source_bin" file="hifiasm-out13.ovlp.source.bin" ftype="binary" />
```

Comment:
@abueg what is `ovlp_source_bin`

We have this now:

```
.. ERROR: Test 14: Found output tag with unknown name [ovlp_source_bin], valid names ['raw_unitigs', 'processed_unitigs', 'primary_contig_graph', 'alternate_contig_graph', 'hap1_contigs', 'hap2_contigs', 'raw_unitigs_trio', 'processed_unitigs_trio', 'hic_pcontig_graph', 'hic_acontig_graph', 'hic_balanced_contig_hap1_graph', 'hic_balanced_contig_hap2_graph', 'hic_raw_initig', 'hap1_contigs_hic', 'hap2_contigs_hic', 'log_file', 'noseq_files', 'bin_files']
```

Should this be the new collection `bin_files`?

Reply:
maybe you can just define collection and a `count` https://docs.galaxyproject.org/en/latest/dev/schema.html#id136
? then you don't need to put the binaries in the repo

Reply:
yay thank you for pointing me to that, i think this works!